src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -353,6 +353,9 @@ static void ClearCounters(UsageModel usage, bool weekly, bool monthly)
             usage.NumberOfForkPullRequestsCheckedOut = 0;
             usage.NumberOfForkPullRequestPulls = 0;
             usage.NumberOfForkPullRequestPushes = 0;
+            usage.NumberOfGitHubPaneHelpClicks = 0;
+            usage.NumberOfWelcomeTrainingClicks = 0;
+            usage.NumberOfWelcomeDocsClicks = 0;
```

Comment:
Oh! How embarrassing! This means these metrics up until now are completely wrong...